### DISCUSSION TITLE
Inbound and Outbound Enhanced CRM tabs

### DISCUSSION THREAD

**Discussion by griff4594** [twilio-professional-services/flex-project-template#491] [Ideas]
Having a different URL or tab show up for inbound calls versus outbound calls.

### Use Case:
I have a company that does a lot of outbound calls and uses a lead portal for those calls, when they call outbound it needs to either stay the same URL that is already active OR be able to set an outbound URL tab because they already have the information up on their screen when they are calling out. When an inbound call comes in, the url should be a inbound URL which can be used to auto query the customer in my CRM. 

### Current action:
Right now if you set the task URL, it will be the same for both inbound and outbound tasks. 

### Proposal:
Extending the config to something like this:
```
      "enhanced_crm_container": {
        "enabled": true,
        "inbound_url": "https://test.com/inbound",
        "should_display_url_when_no_tasks": true,
        "display_url_when_no_tasks": "https://test.com.com/",
        "enable_inbound_url_tab": true,
        "inbound_url_tab_title": "Web Tab Inbound",
        "outbound_url_tab_title": "Web Tab Outbound",
        "enable_outbound_url_tab": true,
        "outbound_url": "https://test.com/outbound"
      },
```

I'm pretty sure this isn't already in the Enhanced CRM window, but I think it would be a good addition. 

Maybe even extend further to identify calls from conversations on inbound and outbound.

**Comment by dremin** [comment]
Hi @griff4594! You might be able to accomplish this without any changes to the feature. You may include task attributes (formatted as `{{task.attribute_name_goes_here}}`) in the configured URL, and they will automatically populate. So, you could use an attribute indicating direction, or you could even put the entire URL in as a task attribute and read it from there.
